vhdbuilder/packer/install-dependencies.sh

```diff
@@ -562,4 +562,17 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
   # update message-of-the-day to start after multi-user.target
   # multi-user.target usually start at the end of the boot sequence
   sed -i 's/After=network-online.target/After=multi-user.target/g' /lib/systemd/system/motd-news.service
-fi
\ No newline at end of file
+
+  # disable and mask all UU timers/services
```

Comment:
_with fire_

Reply:
more comments please about why you're not just removing the pacakge or relying on the configfile. I am not sure this is necessary and makes it a bit harder a customer to re-nable on their own with a daemonset (which you coul argue is a plus or a minus).